signature/jws/jwt.go

```diff
@@ -0,0 +1,109 @@
+package jws
+
+import (
+	"crypto/x509"
+	"fmt"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/notaryproject/notation-core-go/signature"
+)
+
+// remoteSigningMethod wraps the external signer to be a jwt.SigningMethod
+type remoteSigningMethod struct {
+	jwt.SigningMethod
```

Comment:
Do we need to implement `Alg()` method?

Reply:
Good question. No, we don't need. Removed the `Alg()` method.